### DISCUSSION TITLE
Add option to base estimated 1RM off a given number of reps or fewer

### DISCUSSION THREAD

**Discussion by roberthill** [astashov/liftosaur#239] [Ideas]
Since 1RM calculators get more inaccurate the more reps are performed, add an option to specify how many reps (or fewer) should be taken into consideration for 1RM calculations for PRs.

**Comment by yrinkel** [comment]
You mean only take in account sets that have bellow a certain number of reps?
Either way 1RM calculation in Liftosaur is not automatic. You have to program it specifically. 

So you can either only do it for shorter sets or calculate 1RM for all sets with a min number of reps conditional test.

**Reply by roberthill** [reply]
Yep, I'm talking about the PRs screen under exercises, not programming. Would be great if my PR took only X reps of my 10 rep set into consideration so it was more accurate (since 1RM calculators are more inaccurate the farther away you get from 1 rep).

**Reply by yrinkel** [reply]
Unfortunately that doesn’t work like that.  
- 1 RM calculator will increase your 1RM as you can perform more reps with a given weight. If you use less reps than you actually did the calculator will greatly underestimate your 1RM. 
- on top of that, if you really had done less reps you could have lifted more weight. So the 1RM calculator will do even more underestimating there.
